util_test.go

```diff
@@ -343,7 +343,19 @@ func (matcher *orderedTransactionMatcher) NegatedFailureMessage(actual interface
 	)
 }
 
-// Tools for testing the library
+/* Tools for testing the library */
+// Check for damage dealt (if any) by a Pokemon in battle
+func DamageDealt(t []Transaction, p *Pokemon) uint {
```

Comment:
(nit) Let's make this return an int so we use `Equal(x)` instead of `BeEquivalentTo(x)` in the matchers